### ISSUE TITLE
Moving a module to a different location could also change it's definition

### ISSUE THREAD

**Issue by stillalex** [Frege/eclipse-plugin#34]
Using the Frege Eclipse IDE, I can define a module M under package 'org.example1':

```
module org.example1.M where
data M = M
```

Now I want to move it to a different package 'org.example2'.
My options are drag&drop or (right-click) Refactor -> Move. The file moves, but the module definition doesn't change:

```
module org.example1.M where
data M = M
```

It would be nice to have the module definition change from `module org.example1.M` to `module org.example2.M`.

**Comment by Ingo60** [comment]
Yes, it would be good to have such refactoring (and others, as well), but the framework the eclipse plugin is build upon doesn't support that.
